index.js

```diff
@@ -72,24 +105,32 @@
 // ***** Question 6 *****
 
 // *** Uncomment the lines below to test
-// console.log("%cQuestion 6", "color: red")
-
-// const users = [ 
-//   { 
-//     name: "Duane", phones: { cell: "555-123-4567", office: "555-456-7890" }
-//   },
-//   { 
-//     name: "Liza", phones: { cell: "555-234-5678", office: "555-567-1234" }
-//   }
-// ]
-// printNameAndPhones(users)
-// // => "Duane"
-// // => "Cell: 555-123-4567"
-// // => "Office: 555-456-7890"
-// // => "Liza"
-// // => "Cell: 555-234-5678"
-// // => "Office: 555-567-1234"
-// console.log("%c----------", "color: red") 
+console.log("%cQuestion 6", "color: red")
+
+function printNameAndPhones(users){
+  for (let i = 0; i < users.length; i++) {
+    console.log(user.name);
+    console.log(user.phones.cell);
+    console.log(user.phones.office);
+  }
```

Comment:
This `for` loop works for iterating, but in order to access each user, you'd need to use the `i` variable to access an element from the array:

```js
  for (let i = 0; i < users.length; i++) {
     // get each element from the array and assign it to a variable
    const user = users[i];
    console.log(user.name);
    console.log(user.phones.cell);
    console.log(user.phones.office);
  }
```

Javascript also has a couple other methods that make it a little easier.

Here's how it would look with `for..of`: 

```js
  for (let user of users) {
    console.log(user.name);
    console.log(`Cell: ${user.phones.cell}`);
    console.log(`Office: ${user.phones.office}`);
  }
```

Here's how it would look with `forEach`: 

```js
  users.forEach(function(user) {
    console.log(user.name);
    console.log(`Cell: ${user.phones.cell}`);
    console.log(`Office: ${user.phones.office}`);
  })
```